src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -217,7 +217,7 @@ const SubscriberHistory = ({ location, dependencies: { dopplerApiClient } }) =>
                               {campaign.urlImgPreview ? (
                                 <div className="dp-tooltip-container">
                                   <a
-                                    href={`https://reports2.fromdoppler.com/Dashboard.aspx?idCampaign=${campaign.campaignId}`}
+                                    href={`${process.env.REACT_APP_REPORTS_URL}/Dashboard.aspx?idCampaign=${campaign.campaignId}`}
```

Comment:
It would be better to use dependency injection for this kind of things.

By the moment, it looks the same, but eventually (not so far), we will need to refactor this kind of things and it will be a mess.

Reply:
how is it? for understand :)

Reply:
Well, you can check for other usages for `process.env...`

If it is not enough to make the idea clear, please call me.

Reply:
fixed

Reply:
Great! it was simpler than I though 😀